data/kahet/kahet outfits.txt

```diff
@@ -19,17 +19,17 @@ outfit "Ka'het Nullifier"
 	"weapon capacity" -97
 	"gun ports" -1
 	weapon
-		sprite "projectile/ionball"
+		sprite "projectile/ion ball"
```

Comment:
An "Ionball" is quite possibly a thing, especially since all the related effects already have spaces in them. The fact that the effects specify `"ionball ring"` suggests that a) the creator had a spacebar and knew how to use it; and thus b) that the lack of a space between `ion` and `ball` is in fact an intentional part of the name.

Just because a noun happens to include two regular words does not mean that it is actually a generic noun that happened to get an adjective mashed into the front because someone forgot to use a space.